src/bun.js/api/server.zig

```diff
@@ -4704,24 +4703,22 @@ pub fn NewServer(comptime ssl_enabled_: bool, comptime debug_mode_: bool) type {
                         }
                     }
                 }
-            } else if (first_arg.as(Request)) |request_| {
-                existing_request = request_.*;
-            } else {
-                const fetch_error = WebCore.Fetch.fetch_type_error_strings.get(js.JSValueGetType(ctx, arguments[0]));
-                return JSPromise.rejectedPromiseValue(globalThis, ZigString.init(fetch_error).toErrorInstance(globalThis)).asRef();
-            }
 
-            if (existing_request == null) {
                 existing_request = Request{
                     .url = url.href,
                     .headers = headers,
                     .body = JSC.WebCore.InitRequestBodyValue(body) catch unreachable,
                     .method = method,
                 };
+            } else if (first_arg.as(Request)) |request_| {
+                existing_request = request_.*;
```

Comment:
i feel like there are more bugs lurking in here due to not cloning the request like with `.clone`

but i think its better than status quo